azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -144,6 +146,7 @@ private void loadPluginJobTypes(final JobTypePluginSet plugins)
           + " found. Attempt to load.");
       try {
         commonPluginLoadProps = new Props(null, commonLoadPropsFile);
+        this.keyStoreLoadProps = commonPluginLoadProps;
```

Comment:
keyStoreLoadProps name is confusing. I think you should use commonPluginLoadProps itself. It will tell that it has common plugin properties available.

Reply:
I also thought the same, however, commonPluginLoadProps is used across multiple functions, that is why decided to use different name. Besides, it is used only for KeyStore purpose. We can discuss this further if needed.